tests/e2e/index.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from "@playwright/test";
+
```

Comment:
We could add an array of links objects like so:

```suggestion
const links = [
  {
    label: "Datenschutz",
    url: "/datenschutz",
    h1: "Datenschutzerklärung zur Webseite",
  },
  {
    label: "Weitere Informationen",
    url: "/weitere-informationen",
    h1: "Weitere Informationen",
  },
  {
    label: "Barrierefreiheit",
    url: "/barrierefreiheit",
    h1: "Erklärung zur Barrierefreiheit",
  },
  {
    label: "Hilfe und Kontakt",
    url: "/hilfe-und-kontakt",
    h1: "Hilfe und Kontakt",
  },
  { label: "Open Source Code", url: "/open-source", h1: "Open Source Code" },
  { label: "Impressum", url: "/impressum", h1: "Impressum" },
];
```